skatepark/patterns/text-list-with-button.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Title: Text list with button
+ * Slug: skatepark/text-list-with-button
+ * Categories: featured, text
```

Comment:
I'm not sure this needs to be in 'featured' either.

Reply:
I guess it doesn't needs to be. Removed from featured.